docker/.env.docker.prod

```diff
@@ -16,4 +16,4 @@ UPWARD_JS_LOG_URL=1
 CHECKOUT_BRAINTREE_TOKEN=redacted
 GOOGLE_MAPS_API_KEY=redacted
 MAGENTO_BACKEND_EDITION=EE
-ENABLE_EXPRESS_SERVER_COMPRESSION=false
+ENABLE_EXPRESS_SERVER_COMPRESSION=true
```

Comment:
Our official guidance should be to do compression at the web server or proxy layer, not the application layer. Any reason you opted for this config instead of adding compression to nginx?

Reply:
Dev wanted to test it out and have it enabled on our AWS instances which are not served by Nginx (I presume).

Reply:
You are right @tjwiebell. Our app is served using nginx but the compression is not enabled.

For instance, this is a PR instance without 
![image](https://user-images.githubusercontent.com/35203638/107402272-62978500-6ac9-11eb-932c-e1b1162c9c74.png)


And here is the same network req from this PR instance with compression enabled,
![image](https://user-images.githubusercontent.com/35203638/107402177-4e538800-6ac9-11eb-8b23-3902a5123300.png)

@dpatil-magento do you know if we can enable compression at nginx level, if so, how long does it take. Till then this can be a stop-gap solution.

Reply:
Given this change I'm assuming we just deploy these Docker images directly to EBS, so nginx is involved. It sounds like this isn't as straight forward as it should be with the `jwilder/nginx-proxy` container, but is possible. Not going to block this PR, but we should consider doing this at the correct layer if our goal is performance.

Where to add config: https://github.com/nginx-proxy/nginx-proxy#proxy-wide
Sample of config needed: https://github.com/h5bp/server-configs-nginx/blob/master/h5bp/web_performance/compression.conf